src/aml_service/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const yotiRequest = require('../yoti_request');
+const AmlResultClass = require('../yoti_request/aml.result').AmlResult;
+const Payload = require('../yoti_request/payload').Payload;
+
+const AmlResultObj = function(rawResult){
+  this.onPepList = rawResult['on_pep_list'];
```

Comment:
String 'on_pep_list' and others could be extracted as const since this pattern is used elsewhere, and also, the same string reused in the `getData()`

Reply:
In fact these are already declared as a `const` in `yoti_request/aml.result`, I will probably move them inside the `common` file and use them across the package.